app/lib/import_export.rb

```diff
@@ -710,9 +710,11 @@ def extract_assessments_details(patients_identifiers, assessments, fields)
 
     assessments_details = []
     assessments.each do |assessment|
-      assessment_details = assessment.custom_details(fields, patients_identifiers[assessment.patient_id])
+      assessment_details = assessment.custom_details(fields, patients_identifiers[assessment.patient_id]) || {}
       if fields.include?(:symptoms)
         symptom_names_and_labels&.map(&:first)&.each do |symptom_name|
+          next if assessments_hash[assessment[:id]].nil?
```

Comment:
Does `assessments_hash` not get set if `fields.include?(:symptoms)`? If symptoms is not passed then could L716 and L718 cause an exception?

Reply:
If `fields.include?(:symptoms)`is false then  L716 and L718 could not be reached because of L714.